rasa/shared/utils/validation.py

```diff
@@ -67,28 +146,29 @@ def validate_yaml_schema(
             "of your file.".format(str(e))
         )
 
-    try:
-        schema_file = pkg_resources.resource_filename(PACKAGE_NAME, schema_path)
-        schema_utils_file = pkg_resources.resource_filename(
-            PACKAGE_NAME, RESPONSES_SCHEMA_FILE
-        )
-        schema_extensions = pkg_resources.resource_filename(
-            PACKAGE_NAME, SCHEMA_EXTENSIONS_FILE
-        )
+    schema_file = pkg_resources.resource_filename(PACKAGE_NAME, schema_path)
+    schema_utils_file = pkg_resources.resource_filename(
+        PACKAGE_NAME, RESPONSES_SCHEMA_FILE
+    )
+    schema_extensions = pkg_resources.resource_filename(
+        PACKAGE_NAME, SCHEMA_EXTENSIONS_FILE
+    )
 
-        c = Core(
-            source_data=source_data,
-            schema_files=[schema_file, schema_utils_file],
-            extensions=[schema_extensions],
-        )
+    c = Core(
+        source_data=source_data,
+        schema_files=[schema_file, schema_utils_file],
+        extensions=[schema_extensions],
+    )
+
+    try:
         c.validate(raise_exception=True)
     except SchemaError:
         raise InvalidYamlFileError(
-            "Failed to validate yaml file. "
             "Please make sure the file is correct and all "
-            "mandatory parameters are specified; to do so, "
-            "take a look at the errors logged during "
-            "validation previous to this exception."
+            "mandatory parameters are specified. Please take a look at the errors "
+            "found during validation",
```

Comment:
A sentence is starting with "Please" twice in a row, I'd suggest:
```suggestion
            "Please make sure the file is correct and all "
            "mandatory parameters are specified. Here are the errors "
            "found during validation",
```